inc/apirest.class.php

```diff
@@ -55,6 +55,20 @@ public static function getTypeName($nb = 0) {
       return __('Rest API');
    }
 
+   public function prepare() {
+      parent::prepare();
+
+      foreach ($_FILES as $filename => $files) {
```

Comment:
Is this move of fileupload var parsing is needed ?

Reply:
Yes, because the`GLPIUploadHandler::uploadFiles` call document validation, and only validade after load config

Reply:
ok, just a suggestion so, maybe we could move this code to a separate function `manageUploadedFiles` (for example) and call after the initialization.